source/tests/system/nifgen_session_tests.cpp

```diff
@@ -149,7 +152,7 @@ TEST_F(NiFgenSessionTest, InitWithErrorFromDriver_ReturnsUserErrorMessage)
   call_init_with_options(kTestInvalidFgenRsrc, "", "", &initialize_response);
 
   EXPECT_EQ(kInvalidFgenRsrc, initialize_response.status());
-  EXPECT_STREQ(kViErrorFgenResourceNotFoundMessage, initialize_response.error_message().c_str());
+  EXPECT_THAT(initialize_response.error_message().c_str(), HasSubstr(kViErrorFgenResourceNotFoundMessage));
```

Comment:
Note that we're both changing code in this area. Second to commit gets to merge it.

Reply:
I'm going to wait for this last update to go through CI checks, but I'll PM you before I merge just in case you're about to submit as well.